### DISCUSSION TITLE
Other ddd frameworks, libs, ...

### DISCUSSION THREAD

**Discussion by clojj** [knittl/ddd-annotations#9] [General]
Other ddd frameworks, libs, ... may already have similar annotations (or even interfaces).
Did you investigate, or do you know what is out there ?

**Comment by knittl** [comment]
To be honest, I don't know about any other frameworks which might offer similar annotations. Most frameworks I have seen expose base classes which need to be subclassed by your own project. I'd be happy to hear about other libraries/frameworks which offer similar tooling. Let me know, if you have some.

**Reply by clojj** [reply]
This seems interesting:
https://github.com/xmolecules/jmolecules

..but I like your considerations about overlapping annotation names (`@Entity @Repository`) better

**Reply by clojj** [reply]
and there is no "kmolecules" yet :)
